isis/src/base/objs/StretchBlob/StretchBlob.h

```diff
@@ -0,0 +1,61 @@
+#ifndef StretchBlob_h
+#define StretchBlob_h
+/**
+ * @file
+ * $Revision: 1.17 $
+ * $Date: 2010/03/22 19:44:53 $
+ *
+ *   Unless noted otherwise, the portions of Isis written by the USGS are
+ *   public domain. See individual third-party library and package descriptions
+ *   for intellectual property information, user agreements, and related
+ *   information.
+ *
+ *   Although Isis has been used by the USGS, no warranty, expressed or
+ *   implied, is made by the USGS as to the accuracy and functioning of such
+ *   software and related material nor shall the fact of distribution
+ *   constitute any such warranty, and no responsibility is assumed by the
+ *   USGS in connection therewith.
+ *
+ *   For additional information, launch
+ *   $ISISROOT/doc//documents/Disclaimers/Disclaimers.html
+ *   in a browser or see the Privacy &amp; Disclaimers page on the Isis website,
+ *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
+ *   http://www.usgs.gov/privacy.html.
+ */
+
+#include "Blob.h"
+#include "CubeStretch.h"
+
+namespace Isis {
+  class CubeStretch;
+  /**
+   * @brief Blob to store stretch information to cube.
```

Comment:
I think both of your sentences make sense here and I'm not sure I'm understanding the distinction. The `StretchBlob` stores stretch information specifically associated with a Cube, but since it's a Blob also allows you to store that information on the Cube. I updated to use your second phrasing, let me know if that works for you.

Reply:
I updated this again. It's both! So I update it to say so.